docs/blog/2024-11-15-zmk-studio-mvp-ga.mdx

```diff
@@ -0,0 +1,233 @@
+},
+{
+name: "Kinesis",
+img: kinesis,
+darkModeImg: kinesisDarkMode,
+url: "https://kinesis-ergo.com/",
+},
+],
+},
+{
+level: "Gold+",
+vendors: [
+{
+name: "CannonKeys",
+img: cannonKeys,
+darkModeImg: cannonKeysDarkMode,
+url: "https://cannonkeys.com/",
+},
+{
+name: "Keychron",
+img: keychron,
+darkModeImg: keychronDarkMode,
+url: "https://keychron.com/",
+},
+],
+},
+{
+level: "Gold",
+vendors: [
+{
+name: "Little Keyboards",
+img: littleKeyboards,
+darkModeImg: littleKeyboardsDarkMode,
+url: "https://littlekeyboards.com/",
+},
+{
+name: "Keebmaker",
+img: keebmaker,
+darkModeImg: keebmakerDarkMode,
+url: "https://keebmaker.com/",
+},
+],
+},
+{
+level: "Silver",
+vendors: [
+{
+name: "keeb.io",
+img: keebio,
+url: "https://keeb.io/",
+},
+{
+name: "Mode Designs",
+img: mode,
+darkModeImg: modeDarkMode,
+url: "https://modedesigns.com/",
+},
+],
+},
+{
+level: "Bronze",
+vendors: [
+{
+name: "deskhero",
+img: deskHero,
+darkModeImg: deskHeroDarkMode,
+url: "https://deskhero.ca/",
+},
+{
+name: "PhaseByte",
+img: phaseByte,
+url: "https://phasebyte.com/",
+},
+{
+name: "Mechlovin'",
+img: mechlovin,
+darkModeImg: mechlovinDarkMode,
+url: "https://mechlovin.studio/",
+},
+],
+},
+{
+level: "Additional",
+vendors: [
+{
+name: "splitkb.com",
+img: splitkb,
+darkModeImg: splitkbDarkMode,
+url: "https://splitkb.com/",
+},
+{
+name: "keycapsss",
+img: keycapsss,
+darkModeImg: keycapsssDarkMode,
+url: "https://keycapsss.com/",
+},
+{
+name: "mekibo",
+img: mekibo,
+darkModeImg: mekiboDarkMode,
+url: "https://mekibo.com/",
+},
+],
+},
+];
+
+return sponsors.map(g => {
+return (
+
+<div>
+  <h4>{g.level} Level</h4>
+  <div
+    style={{
+      display: "grid",
+      gridTemplateColumns: "repeat(auto-fit, minmax(250px, 1fr))",
+      placeItems: "center",
+      gap: "2em 2em",
+    }}
+  >
+    {g.vendors.map((v) => {
+      return (
+        <a href={v.url}>
+          <ThemedImage
+            alt={v.name}
+            style={{ maxHeight: "76px" }}
+            sources={{
+              light: v.img,
+              dark: v.darkModeImg || v.img,
+            }}
+          />
+        </a>
+      );
+    })}
+  </div>
+</div>
+); }); }())}
+
+## Using ZMK Studio
+
+:::note
+
+Additional changes are needed to add metadata about the keyboard's physical layouts in order to use ZMK Studio. See the documentation on [physical layouts](https://zmk.dev/docs/development/hardware-integration/physical-layouts#optional-keys-property) for more information.
+:::
+
+To use ZMK Studio, you need to have a firmware for your keyboard with the feature enabled, as well as a small keymap change to add an unlock key. See [Building with ZMK Studio](https://zmk.dev/docs/features/studio#building) and [ZMK Studio keymap changes](https://zmk.dev/docs/features/studio#keymap-changes) for more details.
+
+See [Accessing ZMK Studio](https://zmk.dev/docs/features/studio#accessing-zmk-studio) for more details on the browser and desktop options for using the ZMK Studio application.
```

Comment:
I think it would be nice to include a list of the in-tree keyboards which are studio ready here. Then you could say something along the lines of "If you have one of the above keyboards, you can flash it with some new firmware enabling ZMK Studio and use it right now"

Reply:
Not a bad idea. Added by pulling from the aggregated metadata.